packages/react-aria-components/example/index.css

```diff
@@ -175,3 +175,38 @@ html {
     }
   }
 }
+
+.dropzone {
+  border: 2px solid white;
+  margin: 20px;
+  padding: 20px;
+  outline: none;
+
+  &[data-focus-visible][data-focused] {
+    border: 2px dashed blue;
+  }
+
+  &[data-hovered]{
+    border-color: 2px dashed green;
```

Comment:
this isn't valid `border-color`, it should only be a color
however, I think we should drop the hovered style completely, as there is nothing to interact with, so changing the style on hover is misleading. instead, we're using the drop-target to indicate when an interaction has a meaningful result